src/plots/segmentPlot.ts

```diff
@@ -0,0 +1,79 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Plots {
+  export class Segment<X, Y> extends XYPlot<X, Y> {
+    private static _X2_KEY = "x2";
+    private static _Y2_KEY = "y2";
+
+    /**
+     * A Segment Plot displays line segments based on the data.
+     *
+     * @constructor
+     */
+    constructor() {
+      super();
+      this.addClass("segment-plot");
+    }
+
+    protected _createDrawer(dataset: Dataset) {
+      return new Drawers.Segment(dataset);
+    }
+
+    protected _generateAttrToProjector() {
+      var attrToProjector = super._generateAttrToProjector();
+      return attrToProjector;
+    }
+
+    protected _generateDrawSteps(): Drawers.DrawStep[] {
+      return [{attrToProjector: this._generateAttrToProjector(), animator: this._getAnimator(Plots.Animator.MAIN)}];
```

Comment:
Not sure this looks quite right. We should stick with an `Animators.Null` until we get a spec from design for what this should look like.